pkg/ipmi/ipmi.go

```diff
@@ -173,3 +207,44 @@ func (i *IPMI) ShutoffWatchdog() error {
 
 	return nil
 }
+
+func (e *Event) Marshall() []byte {
+	buf := &bytes.Buffer{}
+	if err := binary.Write(buf, binary.LittleEndian, *e); err != nil {
+		log.Fatalf("Can't convert struct %v to bytes", *e)
+	}
+
+	return buf.Bytes()
+}
+
+func (i *IPMI) LogSystemEvent(sel []byte) error {
```

Comment:
I think this should take an event instead as parameter.

Reply:
Updated it.